src/components/MerchSection/MerchSection.tsx

```diff
@@ -1,20 +1,20 @@
 import React from "react";
 import * as S from "./MerchSection.styles";
-import MerchTeamPhoto from "../../images/merch/MerchTeamPhoto.jpeg";
+import MerchCollection from "./Merch";
```

Comment:
Please use the same name as the imported component to avoid confusion. I suggest renaming the `Merch` file to `MerchCollection`